src/main/groovy/org/mockito/release/notes/util/GitHubFetcher.java

```diff
@@ -0,0 +1,95 @@
+package org.mockito.release.notes.util;
+
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.List;
+
+/**
+ * This class contains standard operations for skim over GitHub API responses.
+ */
+public abstract class GitHubFetcher {
```

Comment:
I have strong dislike towards abstract classes because they tend to make the design complicated and harder to grasp. At some point I might change the inheritance to composition :)